setup.cfg

```diff
@@ -53,17 +55,4 @@ collect_ignore = ['setup.py']
 
 [pydocstyle]
 convention = google
-add-ignore = D105, D107, D407, D203, D213, D413, D406, D417
-
-[pylint]
```

Comment:
@pvk-developer @amontanez24 Any idea why this is here? It doesn't seem to be used.

Reply:
I don't recall having `pylint` on `SDV`.

Reply:
it comes from this PR https://github.com/sdv-dev/SDV/commit/6ed01d67a5670b511284b21f94f0ccf712af1a0d